python/lsst/ip/isr/overscan.py

```diff
@@ -342,7 +346,10 @@ def collapseArrayMedian(self, maskedArray):
         fitType = afwMath.stringToStatisticsProperty('MEDIAN')
         for row in integerMI:
             newRow = row.compressed()
```

Comment:
What is this `compressed` returning? How could it be empty? (which is what you check for in the next conditional)

Reply:
`compressed` returns all values of the maskedArray that are not masked.  The case where `len(newRow) == 0` is the case where all elements of this row's overscan are masked, and so the `newRow` is empty.